crates/ruff/src/jupyter/schema.rs

```diff
@@ -0,0 +1,253 @@
+//! The json schema of a jupyter notebook, entrypoint is [JupyterNotebook]
+//!
+//! Generated by https://app.quicktype.io/ from
+//! <https://github.com/jupyter/nbformat/blob/16b53251aabf472ad9406ddb1f78b0421c014eeb/nbformat/v4/nbformat.v4.schema.json>
+//! Jupyter Notebook v4.5 JSON schema.
+//!
+//! Some manual edits
```

Comment:
What manual edits? Could we separate the manual edits into their own commit, so that even if the PR is squashed, we have access to the history here?

Reply:
i've added comments on what i changed